alot/db/manager.py

```diff
@@ -385,6 +385,9 @@ def get_threads(self, querystring, sort='newest_first'):
         assert sort in self._sort_orders
         q = self.query(querystring)
         q.set_sort(self._sort_orders[sort])
+        if exclude_tags:
```

Comment:
I tend to agree. This is why the `x = x or []` pattern is idiomatic. I like the change the way it is actually.

Rant about lists and tuples ahead ;)
A tuple and a list are not the same type and they are for different things. Tuples are heterogeneous, that is, the behavior of the elements only matters by position. Something like `[(1, True), (0, False), (7.2, True)]` is good, but `[(1, True), ('x', 7)]` is not. Lists are homogeneous, it's type (or behavior since python is duck typed) that matters, not position. `[1, 1.5, 7]` is perfectly valid, but `['a', 1, False]` is not.